webapp/public/javascripts/angular-app.js

```diff
@@ -18,18 +18,18 @@ lunchrApp.config(['$stateProvider', '$urlRouterProvider', '$locationProvider',
                 controller: 'UserController'
             })
             .state('users.matching', {
-                url: '/matching',
```

Comment:
It didn't make sense to me to have a URL for this state. By having a URL it means that the user could potentially manually navigate to "/users/matching" even if they hadn't pressed the button to start being matched. Also, the semantics just seemed off having the URL be modified to start being matched and modified again once a match is found.